File: plugins/devkit/devkit-java-tests/testData/inspections/insertCancellationCheckFix/EmptyDoWhileLoop_after.java
package inspections.cancellationCheckInLoops;

import com.intellij.openapi.progress.ProgressManager;
import com.intellij.util.concurrency.annotations.RequiresReadLock;


class Clazz {

    @RequiresReadLock
    public static void foo() {
        int i = 0;
        do {
            ProgressManager.checkCanceled();
            // check comments
        }
        while (i < 100);
    }
}

File: plugins/devkit/devkit-kotlin-tests/testData/inspections/insertCancellationCheckFix/EmptyDoWhileLoop_after.kt
package inspections.insertCancellationCheckFix

import com.intellij.openapi.progress.ProgressManager
import com.intellij.util.concurrency.annotations.RequiresReadLock


@RequiresReadLock
fun main() {
  var i = 0
  do {
      ProgressManager.checkCanceled()
      // comments
  } while (i < 10)
}